doc/developer/guide.md

```diff
@@ -257,6 +257,29 @@ just can't detect them for some reason. In this case, you can just run
 `confluent local services schema-registry start` 3 times, and then everything
 is up.
 
+## Running and connecting to local Materialize
+
+Once things are built and CockroachDB is running, you can start Materialize:
+
+```shell
+bin/environmentd --reset -- --all-features --unsafe mode
+```
+
+This should bootstrap a fresh Materialize instance. Once you see the logline
+"environmentd v<version> listening...", you can connect to the database via:
+
+```shell
+psql -U materialize -h localhost -p 6875 materialize
+```
+
+This uses the internal SQL port, which bypasses authentication. If you wish to
+connect using a system account, you can use the external port with the
+`mz_system` user:
```

Comment:
This is backwards! 6875 is the external port; 6877 is the internal port. The external port just doesn't have authentication enabled in this configuration.

Reply:
Good catch! I edited this line so many times it lost all meaning. Fixed.